README.md

```diff
@@ -2,17 +2,22 @@
 
 The new home of the betanextbest.wellcomecollection.org website.
 
-# client
-
-    cd client
-    npm install
-    npm run compile OR npm run compile:watch (for developing)
-       
-# server
-    
-    cd server
-    npm install
-    npm run watch
-    
+## client
+
+    - `cd client`
```

Comment:
Agreed about lists, but these are code blocks, they don't word independently of each other.
Having them as a code block allows another dev to just copy paste once.

You could have:
<pre>
```bash
cd client
npm install
npm run compile:watch # for local development
# or
npm compile # for once of compile, used on build
```
<pre>

That way you get the benefits of both?